includes/functions.php

```diff
@@ -4119,3 +4119,22 @@ function dokan_get_withdraw_threshold( $user_id ) {
 
     return ( $threshold_day ) ? absint( $threshold_day ) : 0;
 }
+
+/**
+ * Dokan Withdraw Limit Value Validation
+ *
+ * @param mixed $option_value
+ * @param mixed $option_name
+ *
+ * @since DOKAN_LITE_SINCE
```

Comment:
Kindly remove DOKAN_LITE_SINCE and manually put the version here. Currently, the DOKAN_LITE_SINCE  replacement script is not working.